backend/src/main/java/atlantbh/restaurants/controllers/CousineController.java

```diff
@@ -0,0 +1,19 @@
+package atlantbh.restaurants.controllers;
+
+import atlantbh.restaurants.models.Cousine;
+import atlantbh.restaurants.services.CousineService;
+import org.hibernate.service.spi.ServiceException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CousineController extends BaseController<Cousine, CousineService> {
+
+    public ResponseEntity findAll() {
```

Comment:
filter insted od find all

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb